core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java

```diff
@@ -237,26 +235,24 @@ public int getViewId() {
 
    @Override
    public void waitForView(int viewId) throws InterruptedException {
-      waitForView(viewId, 0, TimeUnit.SECONDS);
+      waitForView(viewId, Long.MAX_VALUE, TimeUnit.NANOSECONDS);
```

Comment:
I'm more interested why we need `Long.MAX_VALUE` (which I understand as wait forever right)?